v22.1/security-reference/security-overview.md

```diff
@@ -14,6 +14,8 @@ docs_area: reference.security
 
 [Sign up for a CockroachDB Cloud account!](https://www.cockroachlabs.com/get-started-cockroachdb/)
 
+{% include /v22.1/dedicated-pci-compliance.md %}
```

Comment:
I'm not sure we gain much by externalizing this into an include, now that I see where you are including it. Do you think we need to use this include on other PCI pages that will exist eventually?

Reply:
that is the plan, yes.